beacon-chain/p2p/service.go

```diff
@@ -54,6 +54,11 @@ const lookupLimit = 15
 // maxBadResponses is the maximum number of bad responses from a peer before we stop talking to it.
 const maxBadResponses = 5
 
+// Exclusion list cache config values.
+const cacheNumCounters = 1000
+const cacheMaxCost = 1000
+const cacheBufferItems = 64
+
```

Comment:
Please consider combining into a single `const` (if they have a single comment as a group - they can, probably, be combined)